cellprofiler/modules/exporttospreadsheet.py

```diff
@@ -721,6 +721,23 @@ def prepare_run(self, workspace):
 
         returns False if analysis can't be done
         """
+        maximum_image_sets = 500
+        if workspace.measurements.image_set_count > maximum_image_sets:
+            msg = "You are using ExportToSpreadsheet to export " + str(maximum_image_sets) + " image sets. " \
+                                                                     "Instead we suggest using ExportToDatabase" \
+                                                                     "because ExportToSpreadsheet " \
+                                                                     "may fail on large image sets." \
+                                                                     " Do you want to continue?"
+            import wx
```

Comment:
This will break on headless mode if wx isn't installed. You should whether wx is available before importing like this.